projects/js-packages/publicize-components/src/store/selectors.js

```diff
@@ -36,6 +36,31 @@ export function getMustReauthConnections() {
 		.map( connection => connection.service_name );
 }
 
+/**
+ * Returns a template for linkedIn data, based on the first linkedin account found.
+ *
+ * @param {object} args - Arguments.
+ * @param {boolean} args.forceDefaults - Whether to use default values.
+ * @returns {object} The linkedin account data.
+ */
+export function getLinkedInDetails( { forceDefaults = false } = {} ) {
+	if ( ! forceDefaults ) {
+		const connection = getConnections().find( ( { service_name } ) => 'linkedin' === service_name );
+
+		if ( connection ) {
+			return {
+				name: connection.display_name,
+				profileImage: connection.profile_picture,
+			};
+		}
+	}
+
+	return {
+		name: __( 'Account Name', 'jetpack' ),
+		profileImage: 'https://static.licdn.com/sc/h/1c5u578iilxfi4m4dvc4q810q',
```

Comment:
Maybe this is something we could host in the package directly, to avoid any bad surprises if that URL changes in the future?